_dev/apps/ui/src/store/modules/app/actions.ts

```diff
@@ -105,4 +105,17 @@ export default {
   async [ActionsTypes.TRIGGER_REGISTER_HOOK]({}: Context, hookName: string) {
     return fetchShop('registerHook', {hookName});
   },
+
+  async [ActionsTypes.TRIGGER_REGISTER_HOOKS]({dispatch}: Context) {
+    const getModulesStatus = await dispatch(`app/${ActionsTypes.GET_MODULES_VERSIONS}`, 'psxmarketingwithgoogle', {root: true});
```

Comment:
The action you want to call is in the same namespace `app`, so you do not need to call from the root.

```suggestion
    const getModulesStatus = await dispatch(ActionsTypes.GET_MODULES_VERSIONS, 'psxmarketingwithgoogle');
```
Should work as well